legal-api/src/legal_api/reports/business_document.py

```diff
@@ -438,7 +455,7 @@ def _get_environment():
 
     @staticmethod
     def _get_summary_display_name(filing_type: str, filing_sub_type: str, legal_type: str) -> str:
-        if filing_type == 'dissolution':
+        if filing_type in ['dissolution', 'restoration']:
             return BusinessDocument.FILING_SUMMARY_DISPLAY_NAME[filing_type][filing_sub_type][legal_type]
         else:
```

Comment:
```suggestion
        if filing_type == 'dissolution':
            return BusinessDocument.FILING_SUMMARY_DISPLAY_NAME[filing_type][filing_sub_type][legal_type]
        elif filing_type == 'restoration':
           return BusinessDocument.FILING_SUMMARY_DISPLAY_NAME[filing_type][filing_sub_type]
        else:
```

This way you can change below dict to

```
        'restoration': { 
            'fullRestoration': FULL_RESTORATION_STR,
            'limitedRestoration': LIMITED_RESTORATION_STR,
            'limitedRestorationExtension': LIMITED_RESTORATION_EXTENSION_STR,
            'limitedRestorationToFull': LIMITED_RESTORATION_TO_FULL_STR
```